src/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<!-- Targeting both net7.0 and net472 so that other things such as PS cmdlets can use this
+		<!-- Targeting both net8.0 and net472 so that other things such as PS cmdlets can use this
 		which need to support a wider range of machines -->
-		<TargetFrameworks>net7.0;net472;netstandard2.0</TargetFrameworks>
```

Comment:
We should also be doing a major version bump of this (defined in build-and-release.yml)

Reply:
Do the STS version as well while you're at it